src/components/Company/CompanyName/CorrectCompanyName.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-form ref="correctNameForm" v-model="valid" lazy-validation>
+    <v-layout row>
+      <v-flex>
+        <v-text-field
+          v-model="companyName"
+          filled
+          req
+          persistent-hint
+          :rules="companyNameRules"
+          data-test="business-identifier"
+        ></v-text-field>
+      </v-flex>
+    </v-layout>
+  </v-form>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Watch, Emit, Vue } from 'vue-property-decorator'
+import { Action, Getter, State } from 'vuex-class'
+
+// Interfaces && enums
+import { ActionBindingIF, BusinessInformationIF, NameRequestIF } from '@/interfaces'
+import { CorrectionTypes } from '@/enums'
+
+@Component({})
+export default class CorrectNameRequest extends Vue {
```

Comment:
Should the class name be CorrectCompanyName?

Reply:
Absolutely, thank you 😅

Reply:
I see this is now fixed.